README.md

```diff
@@ -6,7 +6,7 @@ Kogito Operator was designed to deploy [Kogito Runtimes](https://github.com/kieg
 
 ## Requirements
 
-- go v1.12+
+- go v1.12+ (**Note:** The v1.12.7 go tools installed via [dnf](https://developer.fedoraproject.org/tech/languages/go/go-installation.html) on Fedora don't work with operator-sdk v0.9.0. Install with the [archive](https://golang.org/doc/install#install) instead.)
```

Comment:
@xiezhang7 do we have any evidence (besides our tests) of this problem? Maybe a github issue, a mail thread, or anything else in the community about this issue? Would be nice to add a link here to such discussion.